.changeset/ten-scissors-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+Added new feature which is calculating the space above and below the component and based on that the flyout will open to the top or bottom. I also added new 'alignment' property which will align the flyout to the right, left or will set the width to match the width of the input.
```

Comment:
feat(OnyxSelect): add property `alignment` and auto detection for opening to top/bottom depending on the available space